thingif/src/main/java/com/kii/thingif/query/GroupedHistoryStatesQuery.java

```diff
@@ -70,7 +70,7 @@ public Builder setClause(@Nullable QueryClause clause) {
 
         /**
          * Set firmware version for grouped query.
-         * @param firmwareVersion firmware version
+         * @param firmwareVersion older firmware version, which defined this alias.
```

Comment:
the `firmwareVersion` in history query here, basically is for querying states  from earlier firmware version.